errors/error.go

```diff
@@ -0,0 +1,151 @@
+// Copyright 2009 The Go Authors. All rights reserved.
```

Comment:
Can we move this to a constants folder and name it statuses or statusCodes? there are more than just error codes in the file.